src/containers/TurnitinDisplay/components/HyperlinkCell.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Hyperlink } from '@edx/paragon';
+
+export const HyperlinkCell = ({ value }) => (
+  <Hyperlink destination={value} target="_blank">
+    View report
```

Comment:
Add this text to `messages.js`

Reply:
Thank you! I couldn't figure it out before. I already uploaded that suggestion.